app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/triage/singledisposals/asset_type_not_yet_implemented.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.actions.RequestWithSessionData
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.returns.triage
+
+@this(
+mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link
+)
+
+@()(implicit request:RequestWithSessionData[_], messages:Messages, appConfig: ViewConfig)
+
+@title = @{messages("disposalDateMixedUseOrIndirect.title")}
+
+@mainTemplate(title = title, userType = request.userType) {
+
+@backLinkComponent(triage.routes.SingleDisposalsTriageController.whenWasDisposalDate())
```

Comment:
this should point to the asset type page. I'd make the back link a parameter of type `Call` to this template so that we can pass in different values if we reuse this template